python/lsst/cp/pipe/utils.py

```diff
@@ -2088,3 +2092,101 @@ def bin_focal_plane(
         binned["itl"] = np.concatenate(itl_arrays).astype(np.bool_)
 
     return binned
+
+
+def bin_flat(ptc, exposure, bin_factor=8, amp_boundary=20, apply_gains=True, gain_ratios=None):
+    """Bin a flat image, being careful with amplifier edges.
+
+    This will optionally apply gains, and apply any gain
+    ratios.
+
+    Parameters
+    ----------
+    ptc : `lsst.ip.isr.PhotonTransferCurveDatasets`
+        PTC dataset with relevant gains.
+    exposure : `lsst.afw.image.Exposure`
+        Exposure to bin.
+    bin_factor : `int`, optional
+        Binning factor.
+    amp_boundary : `int`, optional
+        Boundary around each amp to ignore in binning.
+    apply_gains : `bool`, optional
+        Apply gains before binning?
+    gain_ratios : `np.ndarray`, optional
+        Array of gain ratios to apply.
+
+    Returns
+    -------
+    binned : `astropy.table.Table`
+        Table with detector positions at the center of each bin
+        (``xd``, ``yd``); average image values (``value``); and amplifier
```

Comment:
If I'm following correctly, these are in full image/detector coordinates.  Can this docstring explicitly note that the `(xd, yd)` positions are in detector coordinates?